tests/stakingRewards/StakingRewards.js

```diff
@@ -346,12 +342,6 @@ contract("StakingRewards", (accounts) => {
                 multiplier.mul(wsIteration).sub(new BN(1)).toString()
             );
             await staking.MOCK_priorWeightedStakeAtBlock(0, stopBlock.toString());
```

Comment:
to be on the safe side ;)
```suggestion
            await staking.MOCK_priorWeightedStakeAtBlock(0, stopBlock.toString());
            await staking.MOCK_priorWeightedStake(0);
```